src/nebula-hooks/use-sorting.js

```diff
@@ -35,4 +37,12 @@ export default function sortingFactory(model) {
 
     model.applyPatches(patches, true);
   };
-}
+};
+
+const useSorting = (model) => {
+  const changeSortOrder = useMemo(() => sortingFactory(model), [model]);
```

Comment:
as `const model: EngineAPI.IGenericObject | undefined`, the model can be undefined. I assume that `if(model)` is needed here

Reply:
hmm maybe... I'll add a sanity check in the factory function

Reply:
done